src/assets/icons/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './RadioButtonChecked';
+export * from './RadioButtonUnchecked';
```

Comment:
This is a helper file so that we can import all of the folder imports from the directory and don't need to write separate imports for each file. Se how we have imported the icons in the radio button react component:
```tsx
import {
  RadioButtonChecked,
  RadioButtonUnChecked,
} from '../../../assets/icons';
```
Notice it is from `../assets/icons` not `../assets/icons/RadioButtonChecked` and `../assets/icons/RadioButtonUnchecked`